Snowflake.Data/Client/SnowflakeDbConnectionPool.cs

```diff
@@ -145,5 +151,15 @@ public static int GetCurrentPoolSize()
         {
             return connectionPool.Count;
         }
+
+        public static void SetPooling(bool isEnable)
+        {
+            pooling = isEnable;
```

Comment:
what is expected behavior when pool was enabled and now user set it disable? Do we need to clear the pooled connections or throw an exception?

Reply:
Yes, I think after we enable or disable the pool, we should clear it. I will update this.